release/pipelines/windows-efi-installer/README.md

```diff
@@ -110,6 +110,47 @@ EOF
 oc create -f - <<EOF
 apiVersion: tekton.dev/v1
 kind: PipelineRun
+metadata:
+    generateName: windows10-installer-run-
+spec:
+    params:
+    -   name: winImageDownloadURL
+        value: ${WIN_IMAGE_DOWNLOAD_URL}
+    -   name: preferenceName
+        value: windows.10
+    -   name: autounattendConfigMapName
+        value: windows10-efi-autounattend
```

Comment:
Please remind me, does the config map contain virtio drives? If so you could use `windows.10.virtio` above.

Reply:
yes, I will fix it